tests/cli.test.ts

```diff
@@ -1,8 +1,21 @@
 import { describe, expect, it } from "vitest";
-import { hello } from "../src/index";
+import path from "node:path";
+import { execaCommandSync, type SyncOptions, type SyncResult } from "execa";
 
-describe("Demo test", () => {
-  it("should return 'Hello World'", () => {
-    expect(hello()).toBe("Hello World");
+const CLI_PATH = path.join(__dirname, "../dist/index.js");
+const project_name = "test-project";
+const generatePath = path.join(__dirname, project_name);
+
+const run = <SO extends SyncOptions>(
+  args: string[],
+  options?: SO
+): SyncResult<SO> => {
+  return execaCommandSync(`node ${CLI_PATH} ${args.join(" ")}`, options);
+};
```

Comment:
Cái generic này có vẻ ko cần thiết, vì lúc em chạy test cũng chỉ là `run([])` rồi thôi.

```suggestion
const fun = (args: string[], options?: SyncOptions) => {
  return execaCommandSync(`node ${CLI_PATH} ${args.join(" ")}`, options);
};
```